.env-kubernetes

```diff
@@ -1,15 +1,22 @@
-export KONFIG_PROJECTROOT=/opt/koding
```

Comment:
we may need to get rid of from this special config file sooner than it might seem ;)